app/templates/route-plans/show/route-visits/show/fulfillments/show/index.hbs

```diff
@@ -1,24 +1,25 @@
 {{ui/drilldown-button
-  icon='assignment_turned_in'
-  label='Track Inventory'
+  icon="assignment_turned_in"
+  label="Track Inventory"
   completed=trackingCompleted
-  onClick=(route-action 'track')}}
+  onClick=(route-action "track")}}
 
 {{ui/drilldown-button
-  icon='touch_app'
+  icon="touch_app"
   disabled=(not trackingCompleted)
-  label='Review & Sign'
+  label="Review & Sign"
   indicateCompleted=signed
-  onClick=(route-action 'review')}}
+  onClick=(route-action "review")}}
 
 <div class="row center toolbarContainer">
   <div class="card-1">
     {{ui/icon-button
-      icon='check'
-      label='Submit'
-      size='2'
+      icon="check"
+      label="Submit"
+      size="2"
       color=white
       backgroundColor=colors.SKY_BLUE
-      onClick=(route-action 'submitFulfillment')}}
+      onClick=(route-action "submitFulfillment")
+      disabled=(not model.pod.valid)}}
```

Comment:
Let's refactor `valid` to `isValid`. Doesn't seem like it's used at all since now so should be simply refactor.